proto/qdrant/cloud/booking/v1/booking.proto

```diff
@@ -114,6 +114,16 @@ enum PackageStatus {
   PACKAGE_STATUS_DEACTIVATED = 2;
 }
 
+// PackageTier defines the valid tiers a package can be in.
+enum PackageTier {
+  // Unspecified package tier.
+  PACKAGE_TIER_STATUS_UNSPECIFIED = 0;
+  // Vanilla standard 2023 tier, which we have ever since Qdrant Cloud.
```

Comment:
What about 
// Standard tier
All other info isn't really needed for public documentation?